hkmc2/shared/src/main/scala/hkmc2/codegen/js/JSBuilder.scala

```diff
@@ -208,19 +208,19 @@ class JSBuilder extends CodeBuilder:
                 val ths = mkThis(owner)
                 fun match
                 case S(f) =>
-                  doc"${ths}.${sym.nme} = ${f}; # ${ths}.${sym.nme}.class = ${clsJS}"
+                  doc"${ths}.${sym.nme} = ${f}; # ${ths}.${sym.nme}.class = ${clsJS};"
                 case N =>
-                  doc"${ths}.${sym.nme} = ${clsJS}"
+                  doc"${ths}.${sym.nme} = ${clsJS};"
               case N =>
                 fun match
-                case S(f) => doc"${f}; # ${sym.nme}.class = ${clsJS}"
+                case S(f) => doc"${f}; # ${sym.nme}.class = ${clsJS};"
                 case N => clsJS
         thisProxy match
-          case S(proxy) => doc" # const $proxy = this; # ${res.stripBreaks};${returningTerm(rst)}"
-          case N => doc"$res;${returningTerm(rst)}"
+          case S(proxy) => doc" # const $proxy = this; # ${res.stripBreaks}${returningTerm(rst)}"
+          case N => doc"$res${returningTerm(rst)}"
```

Comment:
What's the rationale for removing semicolons in two places and adding them back in five places instead?

Reply:
This is to remove the semicolon for class and functions